wiki/Tournaments/Mappool/en.md

```diff
@@ -0,0 +1,95 @@
+---
+stub: true
+tags:
+  - map pool
+  - modpool
+  - mod pool
+  - mappool slot
+  - map pool slot
+  - pool slot
+  - rice
+  - LN
+  - hybrid
+  - SV
+  - no mod
+  - NM
+  - free mod
+  - force mod
+  - forced mod
+  - freemod
+  - forcemod
+  - forcedmod
+  - FM
+  - mixed mod
+  - MM
+  - tiebreaker
+---
+
+# Mappool
```

Comment:
while I'm a total outsider to the tournament scene, isn't a mappool a pool of maps, akin to the news post being a post of, well, news, and should be spelled similarly?